apps/web/playwright/organization/organization-invitation.e2e.ts

```diff
@@ -300,10 +300,9 @@ test.describe("Organization", () => {
 
       await test.step("Signing up with the previous username of the migrated user - shouldn't be allowed", async () => {
         await page.goto("/signup");
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        await expect(page.locator("text=Create your account")).toBeVisible();
```

Comment:
Should we pull in english text from i18n here to add more stability to these tests? i.e. If the text changes, this test doesn't break.

Reply:
Yeah, this makes sense. Will update